app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -153,127 +138,116 @@
+    <string name="song_cache">Cache de Músicas</string>
+    <string name="max_cache_size">Tamanho máximo da cache</string>
+    <string name="unlimited">Ilimitado</string>
+    <string name="clear_all_downloads">Limpar todos os downloads</string>
+    <string name="max_image_cache_size">Tamanho máximo do cache de imagens</string>
     <string name="clear_image_cache">Limpar cache de imagens</string>
     <string name="max_song_cache_size">Tamanho máximo do cache de músicas</string>
-    <string name="clear_song_cache">Clear song cache</string>
+    <string name="clear_song_cache">Limpar a cache de músicas</string>
     <string name="size_used">%s usados</string>
-
     <string name="privacy">Privacidade</string>
-    <string name="pause_listen_history">Pause listen history</string>
-    <string name="clear_listen_history">Clear listen history</string>
-    <string name="clear_listen_history_confirm">Are you sure to clear all listen history?</string>
+    <string name="pause_listen_history">Pausar o histórico</string>
+    <string name="clear_listen_history">Limpar o histórico</string>
+    <string name="clear_listen_history_confirm">Você tem certeza que deseja limpar todo o histórico?</string>
     <string name="pause_search_history">Pausar histórico de pesquisa</string>
     <string name="clear_search_history">Limpar histórico de pesquisa</string>
-    <string name="clear_search_history_confirm">Tem certeza que deseja deletar todo o seu histórico de pesquisa?</string>
-    <string name="enable_kugou">Enable KuGou lyrics provider</string>
-
+    <string name="clear_search_history_confirm">Tem certeza que deseja limpar todo o seu histórico de pesquisa?</string>
+    <string name="enable_kugou">Ativar o provedor de letras \"KuGou\"</string>
     <string name="backup_restore">Backup e restauração</string>
-    <string name="backup">Backup</string>
+    <string name="backup">Fazer backup</string>
     <string name="restore">Restaurar</string>
-    <string name="imported_playlist">Imported playlist</string>
+    <string name="imported_playlist">Playlist importada</string>
     <string name="backup_create_success">Backup criado com sucesso</string>
     <string name="backup_create_failed">Não foi possível criar o backup</string>
     <string name="restore_failed">Falha ao restaurar o backup</string>
-
     <string name="about">Sobre</string>
-    <string name="app_version">Versão do aplicativo</string>
-
-    <string name="new_version_available">New version available</string>
-    <string name="translation_models">Translation Models</string>
-    <string name="clear_translation_models">Clear translation models</string>
-</resources>
+    <string name="app_version">Versão do app</string>
+    <string name="new_version_available">Nova versão disponível</string>
+    <string name="translation_models">Modelos de Tradução</string>
+    <string name="clear_translation_models">Limpar modelos de tradução</string>
```

Comment:
These two translation model strings, I'm a bit confused to be honest.

When I was attempting to pull stuff, I got some reference to the removal of these but afterwards I didn't and unsure what that was about.

Reply:
@mikooomich forgot to mention

Reply:
Ah, that can be removed. We don't have the lyric translations libs anymore.